src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/DashboardClaimStatus.java

```diff
@@ -64,6 +64,9 @@ public enum DashboardClaimStatus {
     REQUESTED_COUNTRY_COURT_JUDGEMENT(
         Claim::claimantRequestedCountyCourtJudgement
     ),
+    DEFAULT_JUDGEMENT(
+        Claim::isClaimantDefaultJudgement
```

Comment:
could you plz add a test for this status ?

Reply:
Added Test Case